### PR TITLE
feat(docs): embed graphiql explorer screencast

### PR DESCRIPTION
**Pull request by amberleyromo**
## Description

Embed GraphiQL explorer screencast in docs.

### CODE REVIEW

File: docs/docs/introducing-graphiql.md [hunk 1]
```diff
@@ -30,6 +30,16 @@ Make sure to check out the GraphiQL docs in the upper right-hand corner of the I
 
 ![A diagram pointing out where to find the GraphiQl docs](images/graphiql-docs.png)
 
+## Using the GraphiQL Explorer
+
+The GraphiQL Explorer enables you to interactively construct full queries by clicking through available fields and inputs without the repetitive process of typing these queries out by hand.
+
+<iframe class="egghead-video" width={600} height={348} src="https://egghead.io/lessons/gatsby-build-a-graphql-query-using-gatsby-s-graphiql-explorer/embed" />
```
Comment (marcysutton) [inline docs/docs/introducing-graphiql.md]:
```suggestion
<iframe class="egghead-video" width={600} height={348} src="https://egghead.io/lessons/gatsby-build-a-graphql-query-using-gatsby-s-graphiql-explorer/embed" title="Video: Build a GraphQL Query using Gatsby’s GraphiQL Explorer" />
```

File: docs/tutorial/part-five/index.md [hunk 2]
```diff
@@ -22,23 +21,25 @@ You can access it when your site's development server is running—normally at
...
-GraphiQL and play with your data! Press <kbd>Ctrl + Space</kbd> (or use <kbd>Shift + Space</kbd> as an alternate keyboard shortcut) to bring up
-the autocomplete window and <kbd>Ctrl + Enter</kbd> to run the GraphQL query. You'll be
-using GraphiQL a lot more through the remainder of the tutorial.
+Poke around the built-in `Site` "type" and see what fields are available on it -- including the `siteMetadata` object you queried earlier. Try opening GraphiQL and play with your data! Press <kbd>Ctrl + Space</kbd> (or use <kbd>Shift + Space</kbd> as an alternate keyboard shortcut) to bring up the autocomplete window and <kbd>Ctrl + Enter</kbd> to run the GraphQL query. You'll be using GraphiQL a lot more through the remainder of the tutorial.
+
+## Using the GraphiQL Explorer
+
+The GraphiQL Explorer enables you to interactively construct full queries by clicking through available fields and inputs without the repetitive process of typing these queries out by hand.
+
+<iframe class="egghead-video" width={600} height={348} src="https://egghead.io/lessons/gatsby-build-a-graphql-query-using-gatsby-s-graphiql-explorer/embed" />
```
Comment (marcysutton) [inline docs/tutorial/part-five/index.md]:
```suggestion
<iframe class="egghead-video" width={600} height={348} src="https://egghead.io/lessons/gatsby-build-a-graphql-query-using-gatsby-s-graphiql-explorer/embed" title="Video: Build a GraphQL Query using Gatsby’s GraphiQL Explorer" />
```

File: docs/docs/introducing-graphiql.md (hunk 1 above)
Comment (amberleyromo) [reply docs/docs/introducing-graphiql.md]: Sounds good! Btw, this is how the other Egghead embeds are on the site rn. I plan to pull this out into an MDX component in the future!

**Comment by marcysutton** [comment]
Thanks, @amberleyromo!

**Pull request merged**